src/app/ReviewService.Abstractions/ReviewConditionBuilder.Extensions.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace ReviewService.Abstractions;
+
+/// <summary>
+/// Extensions for <see cref="IReviewConditionBuilder{TReviewSettings}"/>.
+/// </summary>
+public static partial class ReviewConditionBuilderExtensions
+{
+	/// <summary>
+	/// The number of completed primary actions must be at least <paramref name="minimumActionCompleted"/>.
+	/// </summary>
+	/// <typeparam name="TReviewSettings">The type of the object that we use for tracking.</typeparam>
+	/// <param name="builder"><see cref="IReviewConditionBuilder{TReviewSettings}"/>.</param>
+	/// <param name="minimumActionCompleted">The minimum number of completed actions.</param>
+	/// <returns><see cref="IReviewConditionBuilder{TReviewSettings}"/>.</returns>
+	public static IReviewConditionBuilder<TReviewSettings> WithPrimaryActionCompletedAtLeast<TReviewSettings>(this IReviewConditionBuilder<TReviewSettings> builder, int minimumActionCompleted)
```

Comment:
We should probably remove all the `With` prefixes and perhaps reword the members to make them extra clear.

Reply:
Yes I still need to ask GPT if he's got any ideas.